addons/OPT/$PBOPREFIX$

```diff
@@ -1 +1 @@
-opt\opt\addons\opt
\ No newline at end of file
+@OPT\addons\OPT
```

Comment:
Das ist falsch. Bitte wieder zurück ändern

Reply:
Hattet ihr das woanders schon fest verdrahtet? Und wie kam das merkwürdige Doppel-"opt" zustande?
Ich wollte aber eh noch mal vorschlagen, das wir das besser alles nochmal durch-renamen - Für bessere Trennung zwischen Client- und Servermod.

Reply:
Das ist ein fixes (übliches) Schema der Form `Pfad/Prefix/Mod`

Generelles umbenennen kann man schon machen. Dann aber an allen Stellen. 
So oder so hat sowas aber in diesem PR nix verloren, da das ja ne ganz andere Baustelle ist :)

Reply:
Da der gewählte Namespace als Pfad (oder eher als Symlink) beim Entwickeln mit Live-Patching direkt im ArmA3-Root erscheint, hatte ich mich für ein vorangestelltes mod-typisches "@" entschieden, damit das unscheinbare "opt" nicht mitten im Nirgendwo untergeht.
Ich sehe auch nur Vorteile darin, wenn man den Namespace genauso nennt wie später den Mod-Ordner selbst - Womit wir beim nächsten Thema wären:
- @OPT-Server-Mod
- @OPT-Client-Mod
Werden die Mod-Ordner also später so heißen? Dann ändere ich noch die ganzen Scripte. Und Du müsstest den Pfad hier im Repo anpassen.

Das anscheinend einzige andere Vorkommnis des Namespaces ist (bisher) wohl in der modules.hpp - Dort übrigens mit groß geschriebenem OPT. Das sollte man in diesem Zuge auch direkt korrigieren um Problemen mit case-sensitiven Filesystemen wie z.B. unter Linux aus dem Weg zu gehen.

Reply:
Das mit dem großgeschriebenen Dingern in den modules hatte ich eig mal gefixt gehabt :thinking_man:

Man kann die Dinger schon umbenennen. Aber nochmal: Das Ganze (inklusive der Diskussion darüber) hat hier nix zu suchen. Das ist ne ganz andere Baustelle. 
Kannst gerne n issue dzah aufmachen und dann kann man sich da gesondert drum kümmern

Reply:
Wenn du den Prefix wieder zurück änderst, kann der PR hier gemerged werden :)